### ISSUE TITLE
cloudformation SpotProductDescription default should be Linux/UNIX (Amazon VPC) not Linux/Unix (Amazon VPC)

### ISSUE THREAD

**Issue by mgostick** [cristim/autospotting#216]
# Github issue #

## Issue type ##

- Bug Report

## Build number #

```
nightly-655 
```

## Summary ##

Linux/Unix (Amazon VPC) is incorrect.  It should be Linux/UNIX (Amazon VPC) in the cloudformation template.

## Steps to reproduce ##

To reproduce, just `Launch Stack` using CloudFormation template.

## Expected results ##

Spot's would be launched.

## Actual results ##

Spots would not launch.

```
22:06:29 2018/02/09 22:01:37 instance.go:273: Comparing r3.xlarge with c4.large
22:06:29 2018/02/09 22:01:37 instance.go:116: Comparing price spot/instance:
22:06:29 2018/02/09 22:01:37 instance.go:123: Spot price: 0
22:06:29 2018/02/09 22:01:37 instance.go:124: Instance price: 0.1
22:06:29 2018/02/09 22:01:37 autoscaling.go:885: matt-autospot-test-asg Counting already running spot instances of type r3.xlarge in AZ us-east-1e
22:06:29 2018/02/09 22:01:37 autoscaling.go:896: matt-autospot-test-asg Found 0 r3.xlarge instances
22:06:29 2018/02/09 22:01:37 instance.go:152: Checking current spot quantity:
22:06:29 2018/02/09 22:01:37 instance.go:153: Spot count: 0
...
No cheaper compatible instance type was found, nothing to do here... No cheaper spot instance types could be found
```

**Comment by russellballestrini** [comment]
Yup. Sorry about that!

**Comment by cristim** [comment]
Please also implement validation of the cloudformation stack parameters

**Comment by cristim** [comment]
I fixed it myself in 659f81d979d722ed944208b373e6fb2cbd76d16b, new users should always be able to launch the stack with the default settings.
